examples/05_transformers.ipynb

```diff
@@ -0,0 +1,851 @@
+    "    * [1.2 General transformer signature - simple transformers](#section_1_2)   \n",
+    "    * [1.3 General transformer signature - pairwise series transformers](#section_1_3)         \n",
+    "    * [1.4 General transformer signature - pairwise transformers](#section_1_4) \n",
+    "    * [1.5 Searching for transformers](#section_1_5)         \n",
+    "* [2. Time series transformers in detail](#chapter2)\n",
+    "* [3. Pairwise transformers in detail](#chapter3)  \n",
+    "* [4. Extension guide - implementing your own transformer](#chapter4)        \n",
+    "* [5. Summary](#chapter5)          "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### package imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Overview of transformers in `sktime` <a class=\"anchor\" id=\"chapter1\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section explains the differrent types of transformers found in `sktime`.\n",
+    "\n",
+    "There are four main types of transformation in `sktime`:\n",
+    "\n",
+    "* transforming a series/sequence into scalar- or category-valued features. Examples: `tsfresh`, or extracting `mean` and `variance` overall.\n",
+    "* transforming a series into another series. Examples: detrending, smoothing, filtering, lagging.\n",
+    "* transforming a panel into another panel. Examples: principal component projection; applying individual series transformation to all series in the panel.\n",
+    "* transforming a pair of series into a scalar value. Examples: dynamic time warping distance between series/sequences; generalized alignment kernel between series/sequences.\n",
+    "\n",
+    "Notably, the first three (series to primitive features, series to series, panel to panel) are covered by the same base class template and module. We call these transformers \"simple transformers\", or, simply, \"transformers.\n",
+    "Kernels and distances for time series and sequences have the same mathematical signature and differ only in mathematical properties (e.g., definiteness assumptions) - they are covered by the more abstract scientific type of \"pairwise transformer\".\n",
+    "\n",
+    "Below, we give an overview in sub-sections:\n",
+    "* reviewing common data container formats for series and panels\n",
+    "* showcasing the signature of transformers that transform a series or panel\n",
+    "* showcasing the signature of transformers that transform a pair ot series to a scalar, e.g., distances or kernels\n",
+    "* how to search `sktime` for transformers of a certain type"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1 Data container format<a class=\"anchor\" id=\"section_1_1\"></a>\n",
+    "\n",
+    "`sktime` transformers apply to individual time series and panels (= collections of time series).\n",
```

Comment:
Adding explanation of the term 'instance' that we use later on, and re-writing the stuff in paranthesis as a sentence.

```suggestion
    "`sktime` transformers apply to individual time series and panels. Panels are collections of time series, and we refer to each time series in a panel as an \"instance\" of the panel.\n",
```